internal/vds/vds.h

```diff
@@ -37,26 +37,25 @@ enum interpolation_method {
     TRIANGULAR
 };
 
-struct vdsbuffer metadata(
+struct response metadata(
```

Comment:
I would say that (to me) camel case is more common in C++, but in general it depends on the project. I decided to stick with the lower case/snake case naming for the following reasons:
- We already use lower case/snake case. Sticking with this creates less noise in the PR and it is consistent with what we have in the implementation file as well. 
  
  In the C++ STL I would also expect lower case names rather than upper case. So there are good examples for any capitalization rule.
- The interface between Go and OpenVDS it technically supposed to be C which might have other conventions than C++. At least I have the impression names to be lower case most of the time in C.